commonfate_provider/config/__init__.py

```diff
@@ -79,20 +81,14 @@ def resolve_string(self, p: provider.Provider, key: str, val: provider.String):
                 p.diagnostics.error(f"config {key} is required: {e}")
 
 
+# used only for local provider development.
 DEV_LOADER = Configurer(
     string_loader=loaders.EnvLoader(),
-    secret_string_loader=loaders.DevEnvSecretLoader(),
+    secret_string_loader=loaders.SSMSecretLoader,
 )
-"""
-used only for local provider development.
-"""
```

Comment:
@Eddie023 FYI these docstrings are used so that VS Code shows the comment when you hover over the value (I know they are a bit ugly 🤣)
<img width="438" alt="image" src="https://user-images.githubusercontent.com/17420369/222085113-212d6f60-c6fd-470b-8127-2f60b3e615ed.png">